Resources/public/Admin.js

```diff
@@ -74,7 +74,8 @@ var Admin = {
 
                 select.select2({
                     width: function(){
-                        return Admin.get_select2_width(this.element);
+                        // Select2 v3 and v4 BC. If window.Select2 is defined, then the v3 is installed.
```

Comment:
Will we support v3 for a long time? If we remove support for it in next major, please add a label.

Reply:
>